### PR TITLE
doc(plugins): add ChunksWebpackPlugin

### PR DESCRIPTION
**Pull request by yoriiis**
This PR adds the `ChunksWebpackPlugin` to the plugins documentation page on the community group. The README is automatically fetched from the [GitHub repository](https://github.com/yoriiis/chunks-webpack-plugin).

With `splitChunks.chunks: 'all'` webpack generates chunks that are not known before compilation.
- In a Single Page Application, there is `html-webpack-plugin` to generate styles and scripts chunks dependencies.
- In Multiple Page Application with multiple entry point (e.g. Symfony) this is possible with `chunks-webpack-plugin`.

History: The plugin was already proposed on the official documentation in 2020 (#3771) and  but there was not yet the community group to separate the contributions from the webpack core team.

Note: The latest version of the package and therefore of the readme is not yet available on npm in order to be able to synchronize any feedback made on this MR with the project's readme if necessary.

### CODE REVIEW

**Comment by yoriiis** [comment]
cc @EugeneHlushko and @alexander-akait, we had exchanged at the time in #3771.
_(And much improved the readme by the way 😃)_

**Comment by chenxsan** [comment]
Just copied my comment here https://github.com/webpack/webpack.js.org/pull/6768#issuecomment-1521555785:

> Just a note here, we might want to avoid including any community plugins in our repository in future as we already had the [awesome-webpack](https://webpack.js.org/awesome-webpack/) page reserved for third-party plugins/loaders. So far there's only html-webpack-plugin under the Community group because it's sort of required to use alongside webpack. Fetching too many community plugins in our repository is a burden to our build system, and we can't afford including all third-party plugins even though it's doable technically.

Generally I'm against including any other third-party loaders/plugins in our build since we do not control the contents. Anyone can push something not related in webpack in future after included and we build them into our site without awareness.

**Comment by snitin315** [comment]
Yeah. You can add it on the awesome webpack page.

**Comment by yoriiis** [comment]
@alexander-akait sorry to bother you, do you have an opinion on this addition?

At the time of #3771, we had exchanged and it was in reflexion, the Community section did not exist and that posed problem to differentiate the contents.

Since the addition of this section, from what I understand, it could receive this type of contribution.
For example, `html-wepack-plugin` and `svg-chunk-webpack-plugin` have been released.

If there is a need to add a core team to the project, that is also possible.

**Comment by yoriiis** [comment]
@alexander-akait Any hope on this? Feel free to close the PR otherwise.

**Review by alexander-akait — COMMENTED** [review]
/cc @snitin315

File: src/content/plugins/split-chunks-plugin.mdx
Comment (snitin315) [inline src/content/plugins/split-chunks-plugin.mdx:150]: Can we just link to the github readme docs instead of adding it on webpack.js.org, since it is a 3rd party plugin?
```suggestion
T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/) for Single Page Application or [ChunksWebpackPlugin](https://github.com/yoriiis/chunks-webpack-plugin?tab=readme-ov-file#chunkswebpackplugin) for Multiple Page Application. It will inject all the generated vendor chunks for you.
```

Comment (yoriiis) [reply src/content/plugins/split-chunks-plugin.mdx:150]: Yes of course, I've applied the suggestion

**Comment by yoriiis** [comment]
@snitin315 I've applied your suggestion and the branch is up to date. It looks like the CI needs approval

File: src/utilities/fetch-package-readmes.mjs
Comment (snitin315) [inline src/utilities/fetch-package-readmes.mjs:39]: Do we need this now? We have directly linked to the github readme docs in `split-chunks-plugin.mdx`

Comment (yoriiis) [reply src/utilities/fetch-package-readmes.mjs:39]: It seems interesting to me to keep it, it allows you to directly see who contributed to the project from the plugin's webpack doc.

This is also the case on `svg-chunk-webpack-plugin`.

**Review by snitin315 — APPROVED** [review]
Thank you.

**Pull request merged**
